layersvt/api_dump.h

```diff
@@ -519,18 +524,20 @@ class ApiDumpSettings {
 
     std::ostream &formatNameType(std::ostream &stream, int indents, const char *name, const char *type) const {
         stream << indentation(indents) << name << ": ";
-
         if (use_spaces)
-            stream << std::setfill(' ') << std::setw(name_size - (int)strlen(name) - 2);
+            stream << std::setw(name_size - (int)strlen(name) - 2) << "";
```

Comment:
Your commit message explains this construction, but it's likely a future engineer won't review the commit messages to figure out why this "needless" construction is done.  Add a comment here explaining why the unusual construction is used.

Reply:
Also added comment about that in the other PR.